test/hotspot/jtreg/serviceability/attach/LongArgTest.java

```diff
@@ -27,6 +27,7 @@
  * @bug 8334168
  * @library /test/lib
  * @modules jdk.attach/sun.tools.attach
+ * @build jdk.test.lib.Utils jdk.test.lib.apps.LingeredApp LongArgTest
```

Comment:
Hello @sendaoYan, the test class `LongArgTest` isn't needed in the `@build` set.

Reply:
Thanks for the review. `LongArgTest` has been deleted in `@build` set.